src/syntax-highlighters/monaco/providers/hover-actions.ts

```diff
@@ -0,0 +1,92 @@
+import { Keyword } from "../../../constants/keyword";
+import type { editor, languages, Position } from "monaco-editor";
+import type * as MonacoEditor from "monaco-editor";
+
+export const defaultDocumentationMap = {
+  [Keyword.TYPE]: {
+    summary: `A type or grouping of objects that have similar characteristics. For example:
+- workspace
+- repository
+- organization
+- document`,
+    link: "https://openfga.dev/docs/concepts#what-is-a-type",
+  },
+  [Keyword.RELATIONS]: {
+    summary:
+      "A **relation** defines the possible relationship between an [object](https://openfga.dev/docs/concepts#what-is-an-object) and a [user](https://openfga.dev/docs/concepts#what-is-a-user).",
+    link: "https://openfga.dev/docs/concepts#what-is-a-relation",
```

Comment:
will it make sense if we have `https://openfga.dev/docs` defined as constant and then use do something like 
```
link: `${openFGADocRoot}/concepts#what-is-a-relation`,
```

better yet, `/concepts` and `/configuration-language` also define as constant etc.

Reply:
Considered it, but I don't think it will be that helpful. If someone wants to customize these, they will need to override documentationMap. Just did a change to make that possible through registerDsl.

Reply:
ok